docs/userguide/main.md

```diff
@@ -343,6 +343,15 @@ spec:
         roles:
           - "data"
 ```
+### Dynamic Java heap size in nodePools
```

Comment:
Please do not write a separate heading but integrate this into the previous section "Set Java heap size".